scenarios/chapter2/06_Caverns_of_Saurgath.cfg

```diff
@@ -440,11 +440,11 @@
 
         [message]
             speaker=unit
-            message=_ "You will not get our gold!"
+            message=_ "Ssstay back! A hex be upon all who sssteal the treasure of our emperor!"
```

Comment:
Should he be emperor or king?

Reply:
He can be either one, technically speaking. Which do you prefer?

Reply:
I'm not really sure what suits Saurians best, to be honest

Reply:
Okay, I will just leave it as "emperor" for now since that helps distinguish them more from humans.